src/backend/storeManagers/gog/games.ts

```diff
@@ -148,12 +148,14 @@ interface tmpProgressMap {
   [key: string]: InstallProgress
 }
 
-const defaultTmpProgress = {
-  bytes: '',
-  eta: '',
-  percent: undefined,
-  diskSpeed: undefined,
-  downSpeed: undefined
+function defaultTmpProgress() {
```

Comment:
these default object was inlined before and extracted here, but it needs to be inside a function so we always return a new object

the current code without a function passes the same object around and it's getting updated, it doesn't stay i this initial state